### DISCUSSION TITLE
Multi-user setup

### DISCUSSION THREAD

**Discussion by jp1151419** [UltimaHoarder/UltimaScraper#734] [Q&A] [answered]
How does it manage duplicate content?

I have two accounts, both of which have been subscribed to the same couple free trials but also paid for content on other accounts. If I rip both of them will all the content be downloaded twice or will everything be combined in the same folders?

**Comment by UltimaHoarder** [comment, marked as answer]
Everything is merged, no need to worry about dupes.

**Reply by jp1151419** [reply]
Awesome, thanks!
